tests/test_parameter.py

```diff
@@ -130,11 +134,11 @@ def test_param_limits():
 
 
 def test_overloaded_operators():
-    param_a = ComposedParameter('param_a', lambda: 5 * 4, dependents=None)
-    param_b = ComposedParameter('param_b', lambda: 3, dependents=None)
+    param_a = ComposedParameter('param_a', lambda: 5 * 4, params=None)
```

Comment:
Won't this make problems with the check in BaseComposedParameter for the signature of the value_fn, i.e. I think this issues a Warning. Or is this just for legacy purposes?